### DISCUSSION TITLE
What happend with v-go-back in quasar v2?

### DISCUSSION THREAD

**Discussion by lucho19jose** [quasarframework/quasar#9250] [General - Components / Directives / etc] [answered]
I wanted to use it, but when I go to docs/vue directives, I didn't find go back directive.

**Comment by rstoenescu** [comment, marked as answer]
We can't supply this because of the new Vue 3 architecture. Just use `@click="$router.go(-1)"` instead.

**Reply by lucho19jose** [reply]
hey, @rstoenescu thank you for the answer, router.go(-1) works fine.

**Reply by HevelinFR** [reply]
Thanks
